projects/angular-dev-server/src/lib/getWebpackConfig.ts

```diff
@@ -1,48 +1,59 @@
-import { AngularWebpackPlugin } from "@ngtools/webpack";
-import { Configuration } from "webpack";
+import { AngularWebpackPlugin } from '@ngtools/webpack';
+import { Configuration } from 'webpack';
 
 export async function getWebpackConfig(tmpDir: string): Promise<Configuration> {
+  /* We are using JIT mode as AOT has issues with overriding components (e.g. component providers etc...)
+   * Cf. https://github.com/jordanpowell88/angular-ct/issues/36 */
+  const jitMode = true;
```

Comment:
We need to pass this in as an parameter of this `getWebpackConfig() defaulted to true

Reply:
+1 but we've got to fix AOT first as it produces the following error:
```
class doesn't have @Component decorator or is missing metadata
```

we should probably wait to fix this first.

Reply:
Works for me. We'll just need to create a follow up Issue